workspace.jsonc

```diff
@@ -617,6 +617,11 @@
     "scopes/code": {
       "defaultScope": "teambit.code"
     },
+    "scopes/mdx/readme": {
+      "teambit.envs/envs": {
+        "env": "teambit.envs/env"
+      }
+    },
```

Comment:
when setting `teambit.envs/envs` with the `env` prop you should always set the env itself as a root as well.
it's really important. so it should be like this
```
 "scopes/mdx/readme": {
      "teambit.envs/env": {},
      "teambit.envs/envs": {
        "env": "teambit.envs/env"
      }
    },
```